MAINTAINERS.md

```diff
@@ -86,9 +86,11 @@ To update the lorri version in [nixpkgs][]:
 
    > @GrahamcOfBorg build lorri.tests
 
-2. **`nixos-19.09`**: _after_ the first PR has been merged into `master`,
+2. **latest `nixos` stable**: _after_ the first PR has been merged into `master`,
+   if the new release is not a major version bump (aka a breaking change),
    follow the [backporting procedure][nixpkgs-backporting] to create a PR
-   against `release-19.09`; see for example [NixOS#77432][nixos-stable-pr].
+   against `release-<latest-stable-release-date>` (e.g. `release-20.03`);
```

Comment:
So we no longer update 19.09?

Reply:
Hm, we are halfway into 20.03. But maybe we should?

Reply:
I don’t know if the maintainers accept anything but a patch/security release at this point.

Reply:
I have no opinion on this, but I would like the text to be clear on this point :)

Reply:
Hm, we do try to keep compatibility back with rustc as far as reasonable, but for nixpkgs stable releases I think we can target backports for the latest stable. Those two are not opposed to each other methinks, because you never know which situations people try to get lorri working in. 

e.g. I assume many people are stuck on way old nixpkgs pins in their company repos, so putting lorri in an overlay/flake would still work for them if we make sure we are compatible with older rustc. 

But for NixOS backports, people usually don’t expect any major bumps (and often not even minor bumps, because those could introduce new bugs if they add nontrivial features).

Reply:
Should I add that reasoning to the doc?

Reply:
I think it's not necessary to include all the reasoning, but if I'm a new maintainer for lorri, I'd like to know what I need to do. So I just wanted to clarify that this was what you meant.